app/models/container.rb

```diff
@@ -2,6 +2,7 @@ class Container < ApplicationRecord
   include SupportsFeatureMixin
   include NewWithTypeStiMixin
   include ArchivedMixin
```

Comment:
remove `ArchivedMixin` (since that is included by `ArchivedMixin`) - (5 changes)